src/example-types/filters/tagsQuery.js

```diff
@@ -25,16 +25,14 @@ let addQuotesAndDistance = _.curry((tag, text) => {
 })
 
 // https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_reserved_characters
-let escapeReservedChars = _.flow(
+let replaceReservedChars = _.flow(
   _.toString,
-  // Escape these characters
-  _.replace(/([+\-=&|!(){}[\]^"~*?:\\/])/g, '\\$1'),
-  // Remove these characters
-  _.replace(/[><]/g, '')
+  // Replace characters with white space (\\u0020)
+  _.replace(/([+\-=&|!(){}[\]^"~*?:\\/<>;,$'])/g, '\\u0020')
```

Comment:
`u0020` is unicode for space - we should just replace them with a space character